scrapers/AdultEmpire.yml

```
@@ -102,4 +108,31 @@ xPathScrapers:
                 - regex: ^
                   with: "https://www.adultdvdempire.com"
       URL: //meta[@name='og:url']/@content
-# Last Updated October 09, 2023
+  performerScraper:
+    common:
+      $infoPiece: //*[@id="profileModal"]/div/div/div[2]/div[1]/ul
+    performer:
+      Name: //*[@id="content"]/section/div/div[2]/h1/text()
+      Birthdate: $infoPiecel/li[contains(text(), 'Born:')]/text()
+      Gender:
+        fixed: "Female"
+      Height:
+        selector: $infoPiecel/li[contains(text(), 'Height:')]/text()
+        postProcess:
+          - replace:
+              - regex: ^(\d+).*
+                with: $1
```

Here's that same typo again, and this regex will never match: if the field contains `Height:` it will always be at the start of the line so we need to strip it out. We also convert height to metric because that's what Stash expects
```suggestion
      Height:
        selector: $infoPiece/li[contains(text(), 'Height:')]/text()
        postProcess:
          - replace:
              - regex: Height:\s+(.*)
                with: $1
          - feetToCm: true
```